kinetic/distribution.yaml

```diff
@@ -6834,6 +6834,26 @@ repositories:
       url: https://github.com/timn/ros-rcll_ros_msgs.git
       version: master
     status: developed
+  rdl:
+    doc:
+      type: git
+      url: https://gitlab.com/jlack/rdl.git
+      version: master
+    release:
+      packages:
+      - rdl
+      - rdl_benchmark
+      - rdl_cmake
+      - rdl_dynamics
+      - rdl_urdfreader
+      tags:
+        release: release/kinetic/{package}/{version}
+      url: https://gitlab.com/jlack/rdl_release.git
+      version: 0.9.1-1
+    source:
+      type: git
+      url: https://gitlab.com/jlack/rdl.git
+      version: master    
```

Comment:
The error message also mentions:

> see folded block 'block6' for details

And in block6 it shows a diff in this line. It seems to have trailing whitespaces which should not be there. If you edit the PR and remove those the Travis test will likely pass.